src/utils/analysis-utils.ts

```diff
@@ -648,3 +671,59 @@ export class BaselineLayerResult {
       : `${this.getHazardLayer().title} (${this.statistic})`;
   }
 }
+
+export class PolygonAnalysisResult {
```

Comment:
Quite interesting using OOP here (not really done elsewhere as this project is mostly functional) but all for it if it gets the job done

Reply:
Hope this answers the question.  Please resolve if so.  Thank you.

Reply:
Yup! No need to change any of this I think it's fine, just curious. Thanks for the rational tho, I'll resolve this.